lib/tasks/webpacker/compile.rake

```diff
@@ -1,6 +1,7 @@
 $stdout.sync = true
 
 require "open3"
+require "rbconfig"
```

Comment:
You're right, it's not necessary. I have removed that line and it runs without issue. Created #617 to remove this line.

Thanks!